style.css

```diff
@@ -145,3 +138,110 @@ ul li {
   justify-content: center;
   margin: auto;
 }
+
+.my-work {
+  margin: auto;
+  display: grid;
+  justify-content: center;
+  row-gap: 30px;
+}
+
+.works-head {
+  font-size: 40px;
+  font-weight: 400px;
+  width: 250px;
+  line-height: 52px;
+  overflow-wrap: break-word;
+  text-transform: uppercase;
+  color: var(--secondary-color);
+  margin-bottom: auto;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  gap: 24px;
+}
+
+.my-work-title {
+  font-size: 32px;
+  font-weight: 500;
+  line-height: 44px;
+}
+
+.my-work-description,
+.my-work-tools {
+  font-family: 'Inter', sans-serif;
+}
+
+.my-work-description {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: var(--sixth-color);
+}
+
+.my-work-tools li {
+  border-left: 1px gray solid;
+  padding-left: 15px;
+  margin: 10px;
+}
+
+.organize ul li {
+  display: flex;
+  align-items: center;
+  font-size: 15px;
+  line-height: 36px;
+  font-weight: 400;
+  list-style-type: none;
+  color: var(--fourth-color);
+  width: 100px;
+}
+
+.my-work-tools .li-css {
+  border: none;
+}
+
+.my-work-tools {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+}
+
+.cards ul {
+  margin: auto;
+  font-size: 15px;
+  color: #fff;
+}
+
+.visualize-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 95px;
+  height: 44px;
+  background-color: var(--tenth-color);
+  color: var(--primary-color);
+  font-size: 12px;
+  font-weight: 600;
+  border: none;
+  margin-bottom: 18px;
+  cursor: pointer;
+}
+
+.visualize-button:enabled {
+  background-color: var(--tenth-color);
+}
+
+.visualize-button:hover {
+  background-color: var(--tenth-color);
```

Comment:
Great job for adding effect on the button.
However, here it misses a box-shadow effect to stick to Figma as asked by the previous reviewer.

<img width="643" alt="image" src="https://user-images.githubusercontent.com/7243132/176966931-2d91c9b3-6509-40ed-8bd5-7260810e2e89.png">